dataservice_publisher/app.py

```diff
@@ -44,6 +51,140 @@ async def authenticated(request: web.Request) -> bool:
     return False
 
 
+class WeightedMediaRange:
+    """Class for handling weighted media ranges."""
+
+    def __init__(self, type: str, q: float = 1.0) -> None:
```

Comment:
Kalle denne `media_type`, for å skille den fra `self.type`?
```suggestion
    def __init__(self, media_type: str, q: float = 1.0) -> None:
```

Reply:
Bra intensjon. Endrer til media_range

Reply:
Ja, bra, det er det korrekte navnet! Jeg var bare halvveis der :smile:

Reply:
Fiksa i cb0450451d891dad5df3f0f60339d3913b8f154b